spec/packaging/ibm_cloud_pak/qualification-yaml.md

```diff
@@ -49,8 +49,18 @@ Key name: `security`
       * `Administrator` - The Administrator role of a team that includes the target namespace.
       * `Operator` - The Operator role of a team that includes the target namespace.
 
+#### `security.openshift` Prerequisite Section
+* `openshift`- Describes any Red Hat&reg; OpenShift&reg; requirements.
+  * `securityContextConstraints` - Block: Describes any SecurityContextConstraints requirements.
+    * `name` - String: one of the following pre-defined SecurityContextConstraints names:
+      * `ibm-restricted-scc`
+      * `ibm-anyuid-scc`
+      * `ibm-anyuid-hostpath-scc`
+      * `ibm-anyuid-hostaccess-scc`
+      * `ibm-privileged-scc`
+
 ## Catalog Section
-The catalog section describes how this IBM Cloud Pak should be displayed in a catalog.
+The catalog section describes how this IBM Cloud Pak or IBM Solution Pak should be displayed in a catalog.
 
 For reference, the [IBM Global Catalog](https://console.test.cloud.ibm.com/docs/developing/get-coding/index-rscatalog.html) may be a template for future additions to this section.
```

Comment:
The pointer to IBM Cloud for catalog caught me off guard - how is this template for adding items to section?